CONTRIBUTING.md

```diff
@@ -3,23 +3,23 @@ Contributing to ReadyResponder
 
 First of all: thank you! We appreciate any help you can give ReadyResponder.
 
-You can contribue in several ways:
+You can contribute in several ways:
 
-* Write some code !
-* Report an issue
-* Provide some documentation
-* Make a feature request
-* Share a success story
+* Write some code!
+* Report an issue.
+* Provide some documentation.
+* Make a feature request.
+* Share a success story.
 
 Many people contribute to ReadyResponder by writing some code! Here's how:
 
-1. [Fork](https://help.github.com/articles/fork-a-repo) ReadyResponder
-2. Create a topic branch - `git checkout -b my_branch`
-3. Make your changes
-4. Commit your changes to your branch
-3. Push your branch to your github repo - `git push origin my_branch`
+1. [Fork](https://help.github.com/articles/fork-a-repo) ReadyResponder.
+2. Create a topic branch. - `git checkout -b my_branch`
```

Comment:
Thanks, yes that could be good.
Could we give them a link to github docs, like line 16 ?